src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -38,6 +38,8 @@
     public static final String VALID_TAG_HUSBAND = "husband";
     public static final String VALID_TAG_FRIEND = "friend";
 
+    public static final String EMPTY_ADDRESS = "*No Address Specified*";
```

Comment:
Since the current implementation doesn't allow for an address to be an empty string, I assigned the value "*No Address Specified*" to the address field when the user input doesn't input anything for the address field. Thus, for the test, the input for the address field would be EMPTY_ADDRESS instead of being an empty string 😊

Reply:
I see, yup this makes sense to me too.